src/cli/list.rs

```diff
@@ -75,6 +82,14 @@ pub async fn execute(args: Args) -> miette::Result<()> {
         .environment(&environment_name)
         .ok_or_else(|| miette::miette!("unknown environment '{environment_name}'"))?;
 
+    project
```

Comment:
The lock-file is reload 10 lines below but its actually already returned by this function.

Reply:
Good one!